docs/tools/devtools.mdx

```diff
@@ -50,6 +50,25 @@ Eg.
 }
 ```
 
+Example for Vite + React project:
+
+```ts
+// vite.config.ts
+
+export default defineConfig({
+  plugins: [
+    react({
+      babel: {
+        plugins: [
+          'jotai/babel/plugin-debug-label',
+          'jotai/babel/plugin-react-refresh',
+        ],
```

Comment:
Hi, thanks for the suggestion.
Can you try `jotai/babel/preset` instead? I think it's the way we recommend using the plugins.

Reply:
Hello, @dai-shi. I took it from the other example but sure.

Reply:
Maybe we should fix the other example too??

Reply:
Done